README.md

```diff
@@ -72,7 +72,7 @@ Make sure you have `kind` > 0.3.0 and `kubectl`.
 
 The kubeconfig is on the management cluster in secrets. Grab it and write it to a file:
 
-`kubectl get secrets -o jsonpath='{.data.kubeconfig}' kubeconfig-my-cluster | base64 --decode > ~/.kube/kind-config-my-cluster`
+`kubectl get secrets -o jsonpath='{.data.value}' my-cluster-kubeconfig | base64 --decode > ~/.kube/kind-config-my-cluster`
```

Comment:
By chance can you use `.stringData.value` for the jsonpath and avoid needing to base64 decode it?

Reply:
Not sure I followed. 

But, here are the things that I tried, which all didn't work:
* `kc get secrets -o jsonpath='{.data.value.stringData}' my-cluster-kubeconfig`
* `kc get secrets -o jsonpath='{.stringData.value}' my-cluster-kubeconfig`
the resulting json doesn't have any non-encoded fields with kubeconfig

Reply:
And these are the keys at the top level json
```
kc get secrets -o json my-cluster-kubeconfig | jq keys
[
  "apiVersion",
  "data",
  "kind",
  "metadata",
  "type"
]
```

Reply:
I'm pretty sure `stringData` is a write-only field.

Reply:
From https://kubernetes.io/docs/concepts/configuration/secret/
> The stringData field is provided for convenience, and allows you to provide secret data as unencoded strings.

My interpretation is, you can create secrets by setting either `data` or `stringData`. If you are using `data` the content has to be encoded. Using `stringData`, allows you to use unencoded strings.